decidim-budgets/app/views/decidim/budgets/projects/index.html.erb

```diff
@@ -53,7 +49,7 @@
 
   <section class="layout-main__section">
     <span class="text-xl font-semibold">
-      <%= t("select_projects", scope: "decidim.budgets.projects") %>
+      <%= voting_finished? ? t("selected_projects", scope: "decidim.budgets.projects") : t("select_projects", scope: "decidim.budgets.projects") %>
```

Comment:
Let's remove this whole block is the voting is finished. What it gives you if you select the "Not selected" status is just wrong:

![Screenshot of this title when the "Not selected" filter is selected](https://github.com/decidim/decidim/assets/717367/6074b793-f43a-4f99-adfb-be747fa2f2a6)
 
(Or maybe just showing this title when the filter changes, but I don't remember about any other filter where we're doing that)

Reply:
Done https://github.com/decidim/decidim/pull/11854/commits/a335b290a517861cc167e93532b77bb27233d84f